lib/css/components/_stacks-post-summary.less

```diff
@@ -234,7 +234,8 @@
 
         &.is-pinned {
             color: var(--white);
-            background-color: var(--black-800);
+            background-color: var(--black-700);
+            border-color: var(--black-075);
```

Comment:
```suggestion
```

It was @CGuindon's intention that the entire post summary get outlined in `var(--black-075)`

Reply:
I misunderstood...we do not want the border color on here.